lms/static/scripts/frontend_apps/utils/test/google-api-client-test.js

```diff
@@ -13,6 +13,7 @@ describe('loadLibraries', () => {
 
   afterEach(() => {
     document.createElement.restore();
+    dummyScript.remove();
```

Comment:
This script doesn't get appended to the document, so calling `remove` has no effect. The browser just garbage-collects the DOM element when it is no longer referenced. Calling `remove` doesn't do any harm though.

Reply:
It does get appended:
https://github.com/hypothesis/lms/blob/bd75cd47b757557a887db796da55d969fd48a5e8/lms/static/scripts/frontend_apps/utils/google-api-client.js#L18